lib/game_completion.rb

```diff
@@ -0,0 +1,19 @@
+module TicTacToe
+  class GameCompletion 
+
+    def initialize(console_ui) 
+      @console_ui = console_ui
+    end
+
+    def game_over_messages(game_result, board_array)
+      @console_ui.display_board(board_array)
+      @console_ui.puts_space
+      if game_result == 1 || game_result == 2
```

Comment:
Using integers 0, 1, and 2 to keep track of the game result leads to some confusion for anyone who comes in to read this code.